internal/vds/connection.go

```diff
@@ -124,6 +157,25 @@ func MakeAzureConnection(accounts []string) ConnectionMaker {
 		}
 
 		container, blobPath := splitAzureUrl(blobUrl.Path)
-		return NewAzureConnection(blobPath, container, blobUrl.Host, sanitizeSAS(sas)), nil
+		connection := NewAzureConnection(
+			blobPath,
+			container,
+			blobUrl.Host,
+			sanitizeSAS(sas),
+		)
+
+		/*
+		 * OpenVDS (v3.1.2) segfaults on sas-tokens where the srt-field (allowed
+		 * resource types) contains 'c' (container) but _not_ 'o' (object/blob).
+		 * Such as tokens are valid in a general sense, but does not provide enough
+		 * access in order to read a VDS.
+		 *
+		 * Once this bug is fixed upstream this manuall check can go way
```

Comment:
```suggestion
		 * Once this bug is fixed upstream this manual check can go way
```
- Small typo
- Is there an issue to remind us of removing the manual valiation?

Reply:
typo fixed! No and no, but we should do both

Reply:
Once merged I'll an issue for us with reference to the code

Reply:
Reported upstream and added reminder to remove in our issue-tracker #83